tests/vmi_multus_test.go

```diff
@@ -60,6 +60,12 @@ const (
 	macvtapNetworkConf     = `{"apiVersion":"k8s.cni.cncf.io/v1","kind":"NetworkAttachmentDefinition","metadata":{"name":"%s","namespace":"%s", "annotations": {"k8s.v1.cni.cncf.io/resourceName": "macvtap.network.kubevirt.io/%s"}},"spec":{"config":"{ \"cniVersion\": \"0.3.1\", \"name\": \"%s\", \"type\": \"macvtap\"}"}}`
 )
 
+const (
+	sriovnet1 = "sriov"
```

Comment:
maybe worth to add here 1?

Reply:
This commit has not changed the names of the networks intentionally.
This change is only about keeping the relation between the network names as it is hard to understand what is related to what.

And specifically to your comment, I have not added a following change to adjust this as it does not seem that important. I am good with preserving the current as is.